### PR TITLE
[quantizationTest]: Add quantization for convNetForImageRecognition()

### PR DESCRIPTION
**Pull request by ZchiPitt**
- 1 test from Interpreter/InterpreterAndCPU (871 ms total)
- 1 test from JIT/InterpreterAndCPU (1066 ms total)

### CODE REVIEW

**Comment by opti-mix** [comment]
@ZchiPitt I think the idea was to create a NEW quantized test based on an existing non-quantized test, not to replace the existing test.

**Comment by ZchiPitt** [comment]
Kept the non-quantized version of convNetForImageRecognition() now. There are two tests, 
- one called convNetForImageRecognition()
- the other called convNetForImageRecognition_quantized().

The quantized version runs a bit longer (~100ms)

**Comment by nadavrot** [comment]
@opti-mix @ZchiPitt I think that the test was fine the way that it was before, because you need to be able to train the network somehow.

**Comment by ZchiPitt** [comment]
@nadavrot Hi nadav, either way works for me. So do i change it back to only one test case?

**Comment by nadavrot** [comment]
@ZchiPitt Yes, please. It looked good the way it was before.

**Comment by ZchiPitt** [comment]
@bertmaher It cannot work with CPU backend. Well, initially i was following what the testFindPixelRegression test did. But i also tried CPU backend for my test, it failed with this error: "Cannot select the instruction:
%QuantizationProfile = quantizationprofile @in %gsum3, @inout %histogram, @inout %computationInfo"

**Comment by ZchiPitt** [comment]
@nadavrot  ive changed it back.

File: tests/unittests/MLTest.cpp [hunk 1]
```diff
@@ -881,16 +881,18 @@ static void generateImageData(Tensor &images, Tensor &labels, PseudoRNG &PRNG) {
 
 /// Test the convolutional layer.
 /// Use a simple convnet to learn two classes of images: Line and Cross.
-TEST_P(MLTest, convNetForImageRecognition) {
+/// Add quantization support for the convolution layer test case
```
Comment (nadavrot) [inline tests/unittests/MLTest.cpp]: Period at the end of the sentence. 

 Add quantization support for the convolution layer test case  ->  This test checks the results of the quantized network.

Comment (ZchiPitt) [reply tests/unittests/MLTest.cpp]: done

**Comment by ZchiPitt** [comment]
hi, @nadavrot , sorry i was a bit confused. This convNetForImageRecognition test was initially written for "Use a simple convnet to learn two classes of images: Line and Cross." ?
Im not sure if adding quantization would change the purpose of this test?

**Comment by nadavrot** [comment]
Oh, I am sorry. My mistake. I thought that you’ve quantized the other test. 
LGTM.

**Comment by rdzhabarov** [comment]
" Does the CPU backend not work for quantization profiling or something?" @bertmaher, yup, profiling is running only on interpreter as of now.

**Pull request merged**